src/services/EthService/network/providerHandler.ts

```diff
@@ -92,17 +92,14 @@ export class ProviderHandler {
         const providers = (client as FallbackProvider).providers;
         return any(
           providers.map((p) => {
-            // eslint-disable-next-line no-async-promise-executor
-            return new Promise(async (resolve, reject) => {
-              try {
-                const tx = await p.getTransaction(txhash);
-                // If the node returns undefined, the TX isn't present, but we don't want to resolve the promise with undefined as that would return undefined in the any() promise
-                // Instead, we reject if the tx is undefined such that we keep searching in other nodes
-                return tx ? resolve(tx) : reject();
-              } catch (err) {
-                reject(err);
-              }
-            });
+            // If the node returns undefined, the TX isn't present, but we don't want to resolve the promise with undefined as that would return undefined in the any() promise
+            // Instead, we reject if the tx is undefined such that we keep searching in other nodes
+            return new Promise((resolve, reject) =>
+              p
+                .getTransaction(txhash)
+                .then((tx) => (tx ? resolve(tx) : reject()))
+                .catch((err) => reject(err))
```

Comment:
This `catch` is redundant. If the inner promise rejects, it will reject the outer one automatically.

```suggestion
```